src/main/java/cn/nukkit/command/defaults/EnchantCommand.java

```diff
@@ -123,6 +123,24 @@ public int getIdByName(String value) throws NumberFormatException {
                 return 23;
             case "lure":
                 return 24;
+            case "frost_walker":
+                return 25;
+            case "mending":
+                return 26;
+            case "binding_curse":
+                return 27;
+            case "vanishing_curse":
+                return 28;
+            case "impaling":
+                return 29;
+            case "riptide":
+                return 30;
+            case "loyality":
+                return 31;
+            case "channeling":
+                return 32;
+            case "sweeping":
```

Comment:
sweeping?
```java
    public static final int ID_CROSSBOW_MULTISHOT = 33;
    public static final int ID_CROSSBOW_PIERCING = 34;
    public static final int ID_CROSSBOW_QUICK_CHARGE = 35;
```

Reply:
Umm idk how I managed to get that there...